index.js

```diff
@@ -42,21 +61,32 @@ var apib2swagger = module.exports.convertParsed = function (apib, options) {
         category.content.forEach(function (content) {
             if (content.element === 'resource') {
                 // (name, description) in Resource section are discarded
-                swaggerDefinitions(swagger.definitions, content);
+                const definitions = swaggerDefinitions(content, openApi3);
```

Comment:
Instead of passing in openapi3 here, pass in the entire options variable. This can then be replicated downstream.